newton/tests/test_import_mjcf.py

```
@@ -802,6 +802,43 @@ def test_parse_visuals_vs_hide_visuals(self):
+  <!ENTITY lol2 "&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;">
+  <!ENTITY lol3 "&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;">
+]>
+<mujoco model="bomb">
+  <worldbody>
+    <body name="&lol3;"/>
+  </worldbody>
+</mujoco>
+"""
+
+        try:
+            from defusedxml.common import EntitiesForbidden  # noqa: PLC0415
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)